src/test/java/products/CreateProducts.java

```diff
@@ -0,0 +1,54 @@
+package products;
+import common.BaseURL;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.HashMap;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+
+
+
+public class CreateProducts {
+
+    @Test
+    public void createProduct() {
+        BaseURL.setBaseUrl();
```

Comment:
Želeo sam da primetiš da u svakom testu moraš ručno da uneseš BaseUrl.setBaseUrl. Hajde da razmislimo kako bi ovu metodu mogli da napravimo da se jednom izvrši pre svih testova. Koja bi TestNG anotacija bila dobra za to?